Cargo.toml

```diff
@@ -31,6 +32,7 @@ base64 = "0.22.1"
 serde_json = "1.0.140"
 data-url = "0.3.1"
 regex = "1.12.2"
+time = { version = "0.3.44", features = ["wasm-bindgen"] }
```

Comment:
i don't think `wasm-bindgen` should be enabled by default. This should be only for `wasm-bindgen` environment.

```
[target.'cfg(wasm32)'.dependencies]
time = { version = "0.3.44", features = ["wasm-bindgen"] }
```